vllm/entrypoints/openai/serving_chat.py

```diff
@@ -38,19 +42,19 @@
 
 class OpenAIServingChat(OpenAIServing):
 
-    def __init__(
-        self,
-        async_engine_client: AsyncEngineClient,
-        model_config: ModelConfig,
-        served_model_names: List[str],
-        response_role: str,
-        *,
-        lora_modules: Optional[List[LoRAModulePath]],
-        prompt_adapters: Optional[List[PromptAdapterPath]],
-        request_logger: Optional[RequestLogger],
-        chat_template: Optional[str],
-        return_tokens_as_token_ids: bool = False,
-    ):
+    def __init__(self,
+                 async_engine_client: AsyncEngineClient,
+                 model_config: ModelConfig,
+                 served_model_names: List[str],
+                 response_role: str,
+                 *,
+                 lora_modules: Optional[List[LoRAModulePath]],
+                 prompt_adapters: Optional[List[PromptAdapterPath]],
+                 request_logger: Optional[RequestLogger],
+                 chat_template: Optional[str],
+                 return_tokens_as_token_ids: bool = False,
+                 enable_auto_tools: Optional[bool] = False,
```

Comment:
```suggestion
                 enable_auto_tools: bool = False,
```

Is there a use case for setting `enable_auto_tools=None` instead of the default `enable_auto_tools=False`?

Reply:
11c751dc